accounting/fiscality/taxes/fiscal_positions.rst

```diff
@@ -0,0 +1,110 @@
+==========================================
+Fiscal Positions (tax and account mapping)
+==========================================
+
+Default taxes and accounts are set on products and customers to create new transactions on the fly.
+However, you might have to use different taxes and record the transactions on different accounts,
+according to your customers' and providers' localizations and business types.
```

Comment:
Well spotted, but actually, as they don't share the possession, they both have their own "possessive s". Your suggestion would have been the correct one if the customers and the providers shared what's following.

I had to double-check :D (and I stupidly enjoy reading more about that kind of things, so thanks ^^)
https://grammar.yourdictionary.com/parts-of-speech/nouns/possessive-nouns.html

Reply:
But, don't they share what is following?
oh ok, I got it: customer's localization / provider's business types then.

Ok!

Reply:
No actually, they each have their own type and localization.
But it's not like all customers and all providers share the same localization and type.

It could have been something like this, with your suggestion.
My French customers and providers' main language is French.
They all share that same thing.